packages/test/test-service-load/src/loadTestDataStore.ts

```diff
@@ -584,6 +584,24 @@ class LoadTestDataStore extends DataObject implements ILoadTest {
 		// To avoid growing the file size unnecessarily, not all clients should be sending large ops
 		const maxClientsSendingLargeOps = config.testConfig.content?.numClients ?? 1;
 		let opsSent = 0;
+		let largeOpsSent = 0;
+
+		const reportOpCount = (reason: string, error?: Error) => {
+			config.logger.sendTelemetryEvent(
```

Comment:
Looks like some op metrics are reported in `metricsCleanup` in `runner.ts` as well. Just FYI, in case these are duplicates or maybe they can be merged.